src/Storefront/Controller/ConsignmentController.php

```diff
@@ -124,6 +141,23 @@ public function getForShippingOption(RequestDataBag $request): JsonResponse
             new Context(new SystemSource())
         );
 
+        if (!$consignments->getElements()) {
+            $order = $this->consignmentService->getFullOrderByShippingOptionId($shippingOptionId);
+
+            try {
+                $this->consignmentService->createConsignment(
+                    new Context(new SystemSource()),
+                    $order ?? new OrderEntity(),
```

Comment:
Als er geen order is, hoe kan je dan alsnog zomaar een consignment aanmaken?

Reply:
Dit was om de catch instructie, die hetzelfde is als wanneer er geen order is, maar op 1 plek te hoeven typen. Ik pas het aan.